build.go

```diff
@@ -105,15 +159,22 @@ func Build(procs ProcMgr, logger scribe.Emitter) packit.BuildFunc {
 			return packit.BuildResult{}, fmt.Errorf("failed to copy procmgr-binary into layer: %w", err)
 		}
 
-		processes := []packit.Process{
-			{
-				Type:    "web",
-				Command: "procmgr-binary",
-				Args:    []string{filepath.Join(layer.Path, "procs.yml")},
-				Default: true,
-				Direct:  true,
-			},
+		originalProcess := packit.Process{
+			Type:    "web",
+			Command: "procmgr-binary",
+			Args:    []string{filepath.Join(layer.Path, "procs.yml")},
+			Default: true,
+			Direct:  true,
```

Comment:
Let me make sure I understand:
- if live reload is enabled, the `procs.yml` will contain the reloadable processes (instead of the original)
- the default process type is still `web`, but it contains the watchexec processes?

In most cases we have:
- a separate process for reloadable process types (`Type: "reload-web"`), which is the `default` process
- still have the regular `web` process, but it is NOT the `default`.

Do you think it's possible to mimic that behaviour? If you considered that, what is the benefit of doing this this way?

Reply:
I chose to leave this process definition the same as before since according to the lifecycle, it hasn't changed.

This buildpack has a pretty unique process structure where the entrypoint is another binary that forks out other processes. The `Type` of that entrypoint shouldn't matter as long as it's marked `Default: true`. 

My understanding from https://github.com/buildpacks/spec/blob/main/platform.md is that `Type: web` is only a magic name when API < 0.6, in which case the process with `Type: web` is chosen as the default entrypoint if no other process is marked as default.

Since all of the process reloading is delegated to `procmgr-binary` in this case it made sense to leave the original process definition alone.

Reply:
That seems reasonable to me, thank you for explaining this!